windows/src/desktop/kmshell/xml/strings.xml

```diff
@@ -463,7 +482,7 @@
   <!-- Context: Configuration Dialog - Hotkeys tab -->
   <!-- String Type: PlainText -->
   <!-- Introduced: 7.0.230.0 -->
-  <string name="S_Hotkey_None" comment="Hotkey - text used when no hotkey set for an option">(none)</string>
+  <string name="S_Hotkey_None" comment="Hotkey - text used when no hotkey set for an option">(no hotkey)</string>

```

Comment:
```suggestion
  <string name="S_Hotkey_None" comment="Hotkey - text used when no hotkey set">(no hotkey)</string>
```

I find mentioning of an option a bit confusing here - makes me wonder what other hotkeys there are. Since I assume this string is used for all hotkeys I think we can omit the option.